### DISCUSSION TITLE
How to always show the label of a TextBox and deactivate the animation?

### DISCUSSION THREAD

**Discussion by Stensan** [AvaloniaCommunity/Material.Avalonia#305] [Q&A]
I want the floating label of the TextBox always be at the top. Could you help me with this? I inserted this style in my App.axaml 

```
<Style Selector="TextBox.myTextBox /template/ TextBlock#PART_FloatingWatermark">
      <Setter Property="IsVisible" Value="True" />
</Style>
```

and use Classes="myTextBox" for my TextBox, but it doesn't seem to work. What is the correct selector? I'm pretty new to Avalonia.

`<TextBox Name="Quantity" assist:TextFieldAssist.Label="Quantity" UseFloatingWatermark="True" Classes="myTextBox" Theme="{StaticResource FilledTextBox}" Margin="5"/>`
![276190313-3b6d361e-a8de-4887-9eb1-fecd62b3cd39](https://github.com/AvaloniaCommunity/Material.Avalonia/assets/36670367/2d336f29-a211-41c2-ac17-7804e6e05c80)

**Comment by SKProCH** [comment]
Currently there is no way to do this. As a workaround you can Text=" " (which actually contains space in text.

This is a subject to change in future TextBox rework. You can track progress by this issue: https://github.com/AvaloniaCommunity/Material.Avalonia/issues/280 or in the `textBoxRework` branch.
